api/hearings/data/serviceHearingValues.mock.data.ts

```diff
@@ -67,7 +67,6 @@ export const SERVICE_HEARING_VALUES: ServiceHearingValuesModel = {
       partyID: 'P1',
       partyType: PartyType.IND,
       partyName: 'Jane and Smith',
-      partyChannel: 'byVideo',
```

Comment:
in this mock data, after the partyChannel is removed here, could this be added to the the individualDetails.preferredChannel, this needs replacement not removed entirely.